.github/workflows/dependent-issues.yml

```diff
@@ -6,18 +6,10 @@ on:
     types:
       - opened
       - edited
-      - closed
-      - reopened
-      - synchronize
```

Comment:
I think we should keep these — we need to run on close (so that dependent PRs are unblocked), re-open (so we can easily force an update of the dependencies), and synchronize (so that dependencies are checked on pushes; this isn’t absolutely necessary but it is convenient).

Same goes for `pull_request_target` below.